src/cmd/api/api.py

```diff
@@ -0,0 +1,53 @@
+import bcrypt
+import csv
+
+
+def signUp():
+    username = input("Nieuwe gebruikersnaam: ")
+    password = input("Nieuw wachtwoord: ")
+    with open("./users.csv", 'r') as readFile:
```

Comment:
we're not doing a cli app we are going to be using this logic via an http server so you can go ahead and remove the inputs